static/app/views/performance/browser/webVitals/components/pageOverviewSidebar.tsx

```diff
@@ -114,6 +115,17 @@ export function PageOverviewSidebar({
   const ringSegmentColors = theme.charts.getColorPalette(3);
   const ringBackgroundColors = ringSegmentColors.map(color => `${color}50`);
 
+  const weights =
+    projectScore && containsWeights(projectScore)
+      ? {
+          cls: projectScore.clsWeight,
+          fcp: projectScore.fcpWeight,
+          fid: projectScore.fidWeight,
+          lcp: projectScore.lcpWeight,
+          ttfb: projectScore.ttfbWeight,
+        }
+      : undefined;
+
```

Comment:
I just went ahead and cleaned up a bunch of the types. Part of this involves getting rid of `containsWeights`, because you're right that stored scores always have weights. For frontend calculated scores, they also have hardcoded weights so this check is no longer necessary in either scenario